### PR TITLE
L2D25A3

### PR DESCRIPTION
**Pull request by evyryan**
Added question headers
Fixed fatheads
A few paragraphs that need to be added

### CODE REVIEW

File: kzz7yh-f75957/cod-ve09v2_kzz7yh-f75957.xml [hunk 1]
```diff
@@ -121,7 +122,7 @@
                <p xml:id="kzz7yh-f75957-d1e212">
                   <lb ed="#V" n="64"/>Respondeo quod actus potest dici alicuius potentie 
       <lb ed="#V" n="65"/>dupliciter. Aut quia ab illa elicitur: 
-      <lb ed="#V" n="66"/>cuiusmodi est ille qui ab ea immediate procedit: et hoc 
+      <lb ed="#V" n="66"/>cuius modi est ille qui ab ea immediate procedit: et hoc
```
Comment (RobbieBuhite) [inline kzz7yh-f75957/cod-ve09v2_kzz7yh-f75957.xml:125]:
```suggestion
      <lb ed="#V" n="66"/>cuiusmodi est ille qui ab ea immediate procedit: et hoc 
```

Comment (RobbieBuhite) [reply kzz7yh-f75957/cod-ve09v2_kzz7yh-f75957.xml:125]: These are separate words that are often combined when next to each other.

File: kzz7yh-f75957/cod-ve09v2_kzz7yh-f75957.xml [hunk 2]
```diff
@@ -171,14 +172,15 @@
             </div>
             <div xml:id="kzz7yh-f75957-Dd1e307">
                <head xml:id="kzz7yh-f75957-Hd1e309">Quaestio 2</head>
+               <head type="question-title">Utrum omne ens sit obiectum liberi arbitrii</head>
                <p xml:id="kzz7yh-f75957-d1e311">
                   <lb ed="#V" n="99"/>¶ Questio. II. 
-      <lb ed="#V" n="100"/>SEcundo queritur vtrum omne ens sit 
+      <lb ed="#V" n="100"/>SEcundo queritur vtrum omne ens<!--word--> sit
```
Comment (RobbieBuhite) [inline kzz7yh-f75957/cod-ve09v2_kzz7yh-f75957.xml:178]:
```suggestion
      <lb ed="#V" n="100"/>SEcundo queritur vtrum omne ens sit 
```

Comment (RobbieBuhite) [reply kzz7yh-f75957/cod-ve09v2_kzz7yh-f75957.xml:178]: ens, entis, n., "being, entity," related to ὠν, ὀντος in Greek.

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**